### PR TITLE
[WIP] Fix format

### PR DESCRIPTION
**Pull request by liangxia**
/cc @akostadinov @pruan-rht 

Add the shell script in https://github.com/openshift/verification-tests/pull/1234, this PR is generated with the script via command `tools/formatter.sh $(find . -name *.feature)`

### CODE REVIEW

File: features/build/buildconfig.feature [hunk 1]
```diff
@@ -79,30 +79,30 @@ Feature: buildconfig.feature
     And the "ruby-sample-build-1" build was created
     Then the "ruby-sample-build-1" build completes
 
-    Examples:
-      | template                              |
-      | tc479541/test-buildconfig-s2i.json    | # @case_id OCP-11474
```
Comment (liangxia) [inline features/build/buildconfig.feature:84]: Definitely, this is not something we want.

**Comment by pruan-rht** [comment]
@liangxia nice trick to use `cucumber --dry-run`... but I'm not sure if some of the resulting output is what we want in regards to comments formats...  @akostadinov WDYT?

File: features/build/buildconfig.feature [hunk 2]
```diff
@@ -79,30 +79,30 @@ Feature: buildconfig.feature
...
+    Examples: 
+      | template                           |
+      | tc479541/test-buildconfig-s2i.json |
```
Comment (liangxia) [inline features/build/buildconfig.feature:84]: Definitely, this is not something we want.

Comment (liangxia) [reply features/build/buildconfig.feature:84]: Workarounds,
-  We skip those feature files which contains `Scenario Outline`.
- We change the format to
```
    Examples: 
      | template                           |
      # @case_id OCP-11474
      | tc479541/test-buildconfig-s2i.json |
      # @case_id OCP-xxx
      | tc479541/test-buildconfig-xxx.json |
```

**Comment by rbbratta** [comment]
we could copy  https://github.com/cucumber/cucumber-ruby/blob/v2.4.0/lib/cucumber/formatter/pretty.rb#L47   and modify the indent.

```
                                     FORMAT can also be the fully qualified class name of
                                     your own custom formatter. If the class isn't loaded,
                                     Cucumber will attempt to require a file with a relative
                                     file name that is the underscore name of the class name.
                                     Example: --format Foo::BarZap -> Cucumber will look for
                                     foo/bar_zap.rb. You can place the file with this relative
                                     path underneath your features/support directory or anywhere
                                     on Ruby's LOAD_PATH, for example in a Ruby gem.
```

File: features/build/buildconfig.feature (hunk 2 above)
Comment (akostadinov) [reply features/build/buildconfig.feature:84]: How can we skip scenario outlines?
Changing comment location might be tricky and is uglier IMO. Both are acceptable though.

Or might be easier to just develop out own formatter :) I dunno.

Comment (pruan-rht) [reply features/build/buildconfig.feature:84]: having the comment line in the same place is clearer to me.

**Comment by liangxia** [comment]
> we could copy https://github.com/cucumber/cucumber-ruby/blob/v2.4.0/lib/cucumber/formatter/pretty.rb#L47 and modify the indent.
> 
> ```
> FORMAT can also be the fully qualified class name of
> your own custom formatter. If the class isn't loaded,
> Cucumber will attempt to require a file with a relative
> file name that is the underscore name of the class name.
> Example: --format Foo::BarZap -> Cucumber will look for
> foo/bar_zap.rb. You can place the file with this relative
> path underneath your features/support directory or anywhere
> on Ruby's LOAD_PATH, for example in a Ruby gem.
> ```

Tried,
adding `pretty.rb` to `features/support/ocp/test.rb`
adding `pretty.rb` to `features/support/ocp/test_abc.rb`
adding `pretty.rb` to `features/support/test.rb`
Got,
```
cannot load such file -- ocp/test_abc
Error creating formatter: Ocp::TestAbc (LoadError)
```
```
cannot load such file -- test
Error creating formatter: Test (LoadError)
```

**Comment by akostadinov** [comment]
Do you change class name and module path to match directory path? Sounds like these need to match.

**Pull request closed without merging**